src/Api/navItems.json

```diff
@@ -0,0 +1,4 @@
+[
+  { "id": 1, "title": "Home", "url": "/" },
+  { "id": 2, "title": "Tabs", "url": "/tabs" }
+]
```

Comment:
Can we store such information in the API folder? I thought it's usually used for storing functions and data for working with a server, isn't it?

Reply:
Agree, thank you. I will think about the best place to store this data